server/src/apiClient/SpoonacularClient.js

```diff
@@ -4,23 +4,17 @@ dotenv.config()
 const key = process.env.API_KEY
 
 class SpoonacularClient {
-  static async searchRecipes(searchQuery) {
-    const url = `https://api.spoonacular.com/recipes/complexSearch/?apiKey=${key}&query=${searchQuery}`
-    try {
-      const apiResponse = await got(url)
-      const responseBody = apiResponse.body
-      return responseBody
-    } catch(err) {
-      return { err }
+  static async searchRecipes(searchQuery, targetPage) {
+    let url
+    if(targetPage) {
+      url = `https://api.spoonacular.com/recipes/complexSearch/?apiKey=${key}&query=${searchQuery}&offset=${targetPage*10}`
+    } else {
+      url = `https://api.spoonacular.com/recipes/complexSearch/?apiKey=${key}&query=${searchQuery}`
```

Comment:
nice work! you could also define `baseUrl = https://api.spoonacular.com/recipes/complexSearch/?` as a property on the class, and then call as needed